go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/muhammadmuzzammil1998/jsonc v0.0.0-20201229145248-615b0916ca38
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/rivo/tview v0.0.0-20221029100920-c4a7e501810d
+	github.com/schollz/progressbar/v3 v3.18.0
```

Comment:
If we were adding a download progress bar, I'd like to use this as an opportunity to collaborate with the maintainers behind `briandowns/spinner` which we already use for progress indicators in the GitHub CLI.

To that end, I've reached out as part of https://github.com/briandowns/spinner/discussions/165 to see what makes sense before we make further efforts. 🙇